Automattic/jetpack
# RecordMeterBar Component: Add Accessible Content

This PR adds accessible content to the RecordMeterBar.

#### Changes proposed in this Pull Request:
* Adds a table with the `screen-reader-text` class to output the record meter data as a screen reader accessible table
* Adds `aria-hidden=True` to the record meter and legend graphics so screen readers will only see the screen reader data. 


#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* Run `cd projects/js-packages/storybook && npm run storybook:dev` and confirm that you see RecordMeterBar component in Storybook.
*  Confirm the data table is not displaying on non-screen reader device
*  Confirm the correct data is output in the screen reader only table in the HTML source

![image](https://user-images.githubusercontent.com/30754158/173720664-af48897b-6284-4e76-8f60-96b47c79b200.png)
